### ISSUE TITLE
Not working with Pillow v10

### ISSUE THREAD

**Issue by celunah** [jay3332/pilmoji#29]
The module is not working with Pillow v10 when trying to add text.

This is the error:
```py
Traceback (most recent call last):

  File "/root/venv/lib/python3.11/site-packages/lightbulb/app.py", line 1162, in invoke_application_command
    await context.invoke()

  File "/root/venv/lib/python3.11/site-packages/lightbulb/context/base.py", line 334, in invoke
    await self.command.invoke(self)

  File "/root/venv/lib/python3.11/site-packages/lightbulb/commands/base.py", line 798, in invoke
    await self(context, **kwargs)

  File "/root/venv/lib/python3.11/site-packages/lightbulb/commands/base.py", line 712, in __call__
    return await self.callback(context, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

  File "/root/runner/public/iodine/cogs/leveling.py", line 463, in rank
    name(im, str(member))

  File "/root/runner/public/iodine/cogs/leveling.py", line 425, in name
    pilmoji.text(

  File "/root/venv/lib/python3.11/site-packages/pilmoji/core.py", line 296, in text
    width, _ = font.getsize(content)
               ^^^^^^^^^^^^

AttributeError: 'FreeTypeFont' object has no attribute 'getsize'
```

**Comment by jay3332** [comment]
~~Fixed in #21~~
Ignore this, I only skimmed over the error, sorry.

**Comment by annihilatorrrr** [comment]
> ~Fixed in #21~ Ignore this, I only skimmed over the error, sorry.

Mind making a release if fixed ?

**Comment by jay3332** [comment]
> > ~Fixed in #21~ Ignore this, I only skimmed over the error, sorry.
> 
> Mind making a release if fixed ?

that would be very helpful, thank you!

**Comment by annihilatorrrr** [comment]
> > > ~Fixed in #21~ Ignore this, I only skimmed over the error, sorry.
> > 
> > 
> > Mind making a release if fixed ?
> 
> that would be very helpful, thank you!

I tested a bit everything seems fine now ;)

**Comment by annihilatorrrr** [comment]
> > > ~Fixed in #21~ Ignore this, I only skimmed over the error, sorry.
> > 
> > 
> > Mind making a release if fixed ?
> 
> that would be very helpful, thank you!

Sadly didn't worked :(

**Comment by jay3332** [comment]
It seems like #21 did actually foz this problem, try installing from GitHub and see if the error goes away

**Comment by annihilatorrrr** [comment]
yeah did that only.
  File "C:\Users\kX\PycharmProjects\venv\Lib\site-packages\pilmoji\core.py", line 299, in text
    width, _ = font.getsize(content)
               ^^^^^^^^^^^^
AttributeError: 'FreeTypeFont' object has no attribute 'getsize'

how can if PIL.__version__ >= "9.2.0": this work ? string aff.

Fixed in #30 , please merge it.

**Comment by celunah** [comment]
> yeah did that only. File "C:\Users\kX\PycharmProjects\venv\Lib\site-packages\pilmoji\core.py", line 299, in text width, _ = font.getsize(content) ^^^^^^^^^^^^ AttributeError: 'FreeTypeFont' object has no attribute 'getsize'
> 
> how can if PIL.**version** >= "9.2.0": this work ? string aff.
> 
> Fixed in #30 , please merge it.

Did you verify that your PR fixes issues with #21?

**Comment by annihilatorrrr** [comment]
> > yeah did that only. File "C:\Users\kX\PycharmProjects\venv\Lib\site-packages\pilmoji\core.py", line 299, in text width, _ = font.getsize(content) ^^^^^^^^^^^^ AttributeError: 'FreeTypeFont' object has no attribute 'getsize'
> > how can if PIL.**version** >= "9.2.0": this work ? string aff.
> > Fixed in #30 , please merge it.
> 
> Did you verify that your PR fixes issues with #21?

yes, i tested it, works fine :)

**Comment by celunah** [comment]
> > > yeah did that only. File "C:\Users\kX\PycharmProjects\venv\Lib\site-packages\pilmoji\core.py", line 299, in text width, _ = font.getsize(content) ^^^^^^^^^^^^ AttributeError: 'FreeTypeFont' object has no attribute 'getsize'
> > > how can if PIL.**version** >= "9.2.0": this work ? string aff.
> > > Fixed in #30 , please merge it.
> > 
> > 
> > Did you verify that your PR fixes issues with #21?
> 
> yes, i tested it, works fine :)

Alright, let me test it using my project’s workflow

**Comment by annihilatorrrr** [comment]
> > > > yeah did that only. File "C:\Users\kX\PycharmProjects\venv\Lib\site-packages\pilmoji\core.py", line 299, in text width, _ = font.getsize(content) ^^^^^^^^^^^^ AttributeError: 'FreeTypeFont' object has no attribute 'getsize'
> > > > how can if PIL.**version** >= "9.2.0": this work ? string aff.
> > > > Fixed in #30 , please merge it.
> > > 
> > > 
> > > Did you verify that your PR fixes issues with #21?
> > 
> > 
> > yes, i tested it, works fine :)
> 
> Alright, let me test it using my project’s workflow

yup!

**Comment by celunah** [comment]
> > > > > yeah did that only. File "C:\Users\kX\PycharmProjects\venv\Lib\site-packages\pilmoji\core.py", line 299, in text width, _ = font.getsize(content) ^^^^^^^^^^^^ AttributeError: 'FreeTypeFont' object has no attribute 'getsize'
> > > > > how can if PIL.**version** >= "9.2.0": this work ? string aff.
> > > > > Fixed in #30 , please merge it.
> > > > 
> > > > 
> > > > Did you verify that your PR fixes issues with #21?
> > > 
> > > 
> > > yes, i tested it, works fine :)
> > 
> > 
> > Alright, let me test it using my project’s workflow
> 
> yup!

works just fine, thanks!

maybe you can bump the packages's version to 2.1.0 since it fixed compatibility with Pillow 10

**Comment by annihilatorrrr** [comment]
> > > > > > yeah did that only. File "C:\Users\kX\PycharmProjects\venv\Lib\site-packages\pilmoji\core.py", line 299, in text width, _ = font.getsize(content) ^^^^^^^^^^^^ AttributeError: 'FreeTypeFont' object has no attribute 'getsize'
> > > > > > how can if PIL.**version** >= "9.2.0": this work ? string aff.
> > > > > > Fixed in #30 , please merge it.
> > > > > 
> > > > > 
> > > > > Did you verify that your PR fixes issues with #21?
> > > > 
> > > > 
> > > > yes, i tested it, works fine :)
> > > 
> > > 
> > > Alright, let me test it using my project’s workflow
> > 
> > 
> > yup!
> 
> works just fine, thanks!
> 
> maybe you can bump the packages's version to 2.1.0 since it fixed compatibility with Pillow 10

Dude, after merging https://github.com/jay3332/pilmoji/pull/30 this will work aff :(

**Comment by jay3332** [comment]
Since this was just a bug fix and no features or breaking changes were introduced, I made this a patch update @ [pilmoji v2.0.3](https://pypi.org/project/pilmoji/2.0.3/)
